src/main/java/com/WebController/EventController.java

```diff
@@ -24,24 +24,6 @@
     @Autowired
     private PatientRepository patientRepository;
 
-    @Autowired
-    private AssociationRepository associationRepository;
-
-    /**
-     * DoGet handler for all events.
-     * User will get a list of all existing events with number of attended patients
-     */
-    @GetMapping(value = "/allevents")
-    public Iterable<Event> getAllEvents (@RequestParam("page") int pageNum){
-        Page<Event> events = this.eventRepo.findAll( PageRequest.of(pageNum-1, 10));
-        return events;
-    }
-
-    @GetMapping(value="/allverifiedevents")
-    public Page<Event> getEvents (@RequestParam("page") int pageNum){
-        return this.associationRepository.findAllValidEvents(PageRequest.of(pageNum-1, 10));
-    }
-
     @PostMapping (value = "/eventinfo")
```

Comment:
I think we can even get rid of this method. As long as we have the id of the patient, and an event to add, we should be able to use the CrudRepo for the AssociatioRepo.

Reply:
Those two methods already deleted.